Library/Formula/with-readline.rb

```diff
@@ -23,6 +23,6 @@ def install
   end
 
   test do
-    system "echo 'exit' | #{bin}/with-readline /usr/bin/expect"
+    system "echo", "exit", "|", "#{bin}/with-readline", "/usr/bin/expect"
```

Comment:
This might actually break the test; you'll probably need to use `pipe_output` instead.